iotdb-core/datanode/src/main/java/org/apache/iotdb/db/pipe/connector/protocol/thrift/async/IoTDBThriftAsyncConnector.java

```diff
@@ -176,13 +171,15 @@ public void transfer(TabletInsertionEvent tabletInsertionEvent) throws Exception
       return;
     }
 
+    final long commitId = ((EnrichedEvent) tabletInsertionEvent).getCommitId();
+
     if (isTabletBatchModeEnabled) {
-      final long requestCommitId = commitIdGenerator.incrementAndGet();
-      if (tabletBatchBuilder.onEvent(tabletInsertionEvent, requestCommitId)) {
+      if (tabletBatchBuilder.onEvent(tabletInsertionEvent)) {
         final PipeTransferTabletBatchEventHandler pipeTransferTabletBatchEventHandler =
             new PipeTransferTabletBatchEventHandler(tabletBatchBuilder, this);
 
-        transfer(requestCommitId, pipeTransferTabletBatchEventHandler);
+        transfer(commitId, pipeTransferTabletBatchEventHandler);
```

Comment:
Can all those "commitId"s packed into the its event?

Reply:
I think this may be inconvenient

Reply:
This may have a point..